NST/src/main/java/com/ebay/service/logger/formats/loader/CustomLoggerFormatManager.java

```diff
@@ -94,7 +94,8 @@ private void init() {
 		platformWriters.put(Platform.ANDROID, new AndroidLogger());
 
 		String customLoggerPackage = RuntimeConfigManager.getInstance().getCustomLoggerFormatPackage();
-		if (customLoggerPackage == null) {
+		Boolean useNstDefaultLogger = RuntimeConfigManager.getInstance().getUseNstDefaultLogger();
+		if (customLoggerPackage == null || useNstDefaultLogger) {
```

Comment:
NPE possible here

Reply:
Technically not possible because of how the argument is parsed: https://github.com/eBay/NSTSuite/blob/byarger/loggeroverride/NST/src/main/java/com/ebay/runtime/arguments/UseNstDefaultMockLogger.java#L27

There is no override so it is impossible for it to actually be set to null.

I will add a comment to the code.